src/endpoints/user-status/domain/userstatus.controller.ts

```diff
@@ -0,0 +1,87 @@
+import {
+	DacDirectory,
+	IndexWorldsContract,
+} from '@alien-worlds/eosdac-api-common';
+import { Failure, injectable, Result } from '@alien-worlds/api-core';
+import { config } from '@config';
+import { GetUserStatusInput } from './models/get-user-status.input';
+import { GetUserStatusOutput } from './models/get-user-status.output';
+import { inject } from 'inversify';
```

Comment:
We are trying to depend more on the api-core and less on external packages.

TLDR; please import this from `@alien-worlds/api-core` instead of `inversify` everywhere.